extras/dagger/main.go

```diff
@@ -374,6 +374,9 @@ func (att *Attestation) Push(
 	// The passphrase to decrypt the private key
 	// +optional
 	passphrase *dagger.Secret,
+	// Whether not fail if the policy check fails
+	// +optional
+	exceptionBypassPolicyCheck *bool,
```

Comment:
thanks @javirln for checking, so just so I understand moving forward. Is it required to have the +optional annotation in front of each optional value?

Reply:
As far as I have it understood, yes, see the docs here: https://docs.dagger.io/api/arguments/#optional-arguments
```
Function arguments can be marked as optional. In this case, the Dagger CLI will not display an error if the argument is omitted in the function call.
```